Sources/Danger/Plugins/SwiftLint/SwiftLintViolation.swift

```diff
@@ -20,7 +20,8 @@ public struct SwiftLintViolation: Decodable {
     }
 
     public func toMarkdown() -> String {
-        let formattedFile = file.split(separator: "/").last! + ":\(line)"
+        let filename = file.split(separator: "/").last ?? "unknown"
+        let formattedFile = filename + ":\(line)"
```

Comment:
I was thinking that we could use an empty string instead, given is not relevant for that error.

```suggestion
        let formattedFile = file.split(separator: "/").last.map { "\($0):\(line)"  } ?? ""
```